main.py

```diff
@@ -43,15 +45,10 @@ def update_kontaktdaten_interactive(
 
     with open(filepath, 'w', encoding='utf-8') as file:
         if "plz_impfzentren" not in kontaktdaten:
-            print(
-                "Mit einem Code kann in mehreren Impfzentren gleichzeitig nach einem Termin gesucht werden.\n"
-                "Eine Übersicht über die Gruppierung der Impfzentren findest du hier:\n"
-                "https://github.com/iamnotturner/vaccipy/wiki/Ein-Code-fuer-mehrere-Impfzentren\n\n"
-                "Trage nun die PLZ deines Impfzentrums ein. Für mehrere Impfzentren die PLZ's kommagetrennt nacheinander.\n"
-                "Beispiel: 68163, 69124, 69469\n")
+            print(i18n.t("i18n.InputPLZs"))
```

Comment:
Bei einigen Outputs waren absichtlich Newlines am Ende. Die kann man mit einem leeren `print()` zurückholen

```suggestion
            print(i18n.t("i18n.InputPLZs"))
            print()
```